ftest.sh

```diff
@@ -66,6 +66,26 @@ restore_dist_files() {
 
 }
 
+# For nodes that are only rebooted between CI nodes left over mounts
+# need to be cleaned up.
+pre_clean () {
+    i=5
+    while [ $i -gt 0 ]; do
+        if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh \
+             -S -w "$(IFS=','; echo "${nodes[*]}")" "set -x
+        mapfile -t ftest_mounts < <(grep 'added by ftest.sh' /etc/fstab)
+        for n_mnt in \"${ftest_mounts[@]}\"; do
```

Comment:
(lint) Argument mixes string and array. Use * or separate argument. [SC2145]
(lint) ftest_mounts is referenced but not assigned. [SC2154]